bundles/org.openhab.binding.dsmr/src/main/java/org/openhab/binding/dsmr/internal/device/cosem/CosemObjectFactory.java

```diff
@@ -67,12 +72,16 @@ public CosemObjectFactory() {
          * correct OBISIdentifier is discovered for a certain OBISMsgType this is added to the obisLookupTableDynamic.
          */
         obisLookupTableFixed = new HashMap<>();
+        obisLookupTableMultipleFixed = new HashMap<>();
```

Comment:
Since the maps are final anyway, Why don't you initialize them in the decaration?

Reply:
Good question. I don't know why.  I'll change it.